lib/graph/helpers/ui5Framework.js

```diff
@@ -363,13 +365,20 @@ export default {
 			});
 		}
 
+		const config = await Configuration.fromFile();
+		// ENV var should take precedence over the dataDir from the configuration.
+		const ui5HomeDir = process.env.UI5_DATA_DIR ?
+			path.resolve(path.join(process.env.UI5_DATA_DIR, ".ui5")) :
```

Comment:
I would expect that we use `UI5_DATA_DIR` as the "home dir" and don't create a `.ui5` directory inside of it.

Reply:
Done